openid-sharedsignals-framework-1_0.md

```diff
@@ -822,10 +822,12 @@ events_requested
 
 events_delivered
 
-> **Transmitter-Supplied**, An array of URIs which is the intersection of
-  "events_supported" and "events_requested". These events MAY be delivered over
-  the Event Stream. A Receiver MUST rely on the values received in this field
-  to understand which event types it can expect from the Transmitter.
+> **Transmitter-Supplied**, An array of URIs identifying the set of events
+  which is the intersection of "events_supported" and "events_requested". A 
+  transmitter MAY decide to deliver only a subset of events represented by 
+  this intersection. These events MAY be delivered over the Event Stream. A 
+  Receiver MUST rely on the values received in this field to understand which 
+  event types it can expect from the Transmitter.
```

Comment:
Suggested wording:
events_delivered

**Transmitter-Supplied**: An array of URIs identifying the set of events that the Transmitter SHALL include in the stream. This is a subset of the intersection of "events_supported" and "events_requested". A Receiver MUST rely on the values received in this field to understand which event types it can expect from the Transmitter.

Reply:
> This is a subset of the intersection of "events_supported" and "events_requested"

I feel the above statement suggests that it will always be a subset, which may not be true. 

If you are referring to "A set is a subset of itself" principle to describe this, I feel it may not be clear to an ordinary reader.

Reply:
If there's a mathematical term to specify the subset that isn't the "proper subset", then we can include that. In normative text, I prefer precision, but happy to consider opinions of other members. Perhaps we can say: "This is a subset (not necessarily a proper subset) of ..."